packages/gestalt/src/List/ListItem.js

```diff
@@ -0,0 +1,53 @@
+// @flow strict
+import { type Element, type Node } from 'react';
+import classnames from 'classnames';
+import Text from '../Text.js';
+import ListText from '../ListText.js';
+import styles from '../List.css';
+import getChildrenToArray from '../List/getChildrenToArray.js';
+import { useList } from '../contexts/ListProvider.js';
+
+type Props = {|
+  /**
+   * The list item nested children. See [subcomponents](/web/list#Subcomponents).
```

Comment:
This isn't clear, and the link isn't helpful. Consider something like:
"Used for a nested list, if needed. See the [nesting variant](https://gestalt.pinterest.systems/web/list#Nesting) for more details."